precompiles/addr/Addr.sol

```diff
@@ -16,6 +16,10 @@ interface IAddr {
         string memory customMessage
     ) external returns (string seiAddr, address evmAddr);
 
+    function associatePubKey(
+        string memory pubKeyHex
```

Comment:
nit: it would be good to add some documentation to clarify if this is a compressed or uncompressed pubkey